pom.xml

```diff
@@ -125,7 +125,7 @@
         <maven.compiler.release>8</maven.compiler.release>
 
         <!-- Databases -->
-        <hikaricp.version>3.4.5</hikaricp.version>
+        <hikaricp.version>4.0.3</hikaricp.version>
```

Comment:
:worried: 

Do we need to pickup this version upgrade now. 
There are already a lot of changes in this PR. Let's skip this if we absolutely do not need it for this PR.

Reply:
have done extensive testing